### ISSUE TITLE
In Evalutate Tab, include the Output metrics 

### ISSUE THREAD

**Issue by mabramsKrasdale** [dotnet/machinelearning-modelbuilder#2778]
**Is your feature request related to a problem? Please describe.**
I am watching a youtube video on the use of ML.Net Model Creator from 4 years ago (https://www.youtube.com/watch?v=CnmmhxGUgqg&list=PLdo4fOcmZ0oUDTvk5XMNues09FnuB_D0u&index=8).  In that video it shows the evaluate screen.  On that screen is a section named "Output".  It shows the ML Task, Dataset, Column to predict, Best model, best model Quality (RSqaured), Training time, and Models explored (total).  However, in the VS 2022 (current) version of Model Creator the Evaluate screen only shows "Best Model" with the RMSE and the Model.  It shows no other information.  This information is useful and should be included in the Evaluate screen so that the developer has some idea how accurate the model is after training.  

**Describe the solution you'd like**
I would like to see this information restored.  If the information is not pertinent for any reason, instead of hiding the metrics just show it with "N/A" if that is the case.

**Describe alternatives you've considered**
Show that information on a separate screen, but provide a link to it from the Evaluate screen.

**Additional context**
Current version:
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/146109887/f79e7023-5005-4b07-8f5e-f765291ad511)

Previous Version:
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/146109887/a547c7c4-3a15-441f-b2cd-b1958b2969ab)

**Comment by mabramsKrasdale** [comment]
Ok... in case this puzzles anyone else, I found that different scenario options apparently display different output in the Training tab.  When I selected Data Classification, and ran the training, the result was a different view of the results.

![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/146109887/5afed775-5976-47b6-9b1c-b74ef0c27aad)

So I guess this issue can be put to bed.  Was confusing, but I suppose it makes sense after all.
